layout/AddressesUsingIdsQuery.js

```diff
@@ -1,6 +1,41 @@
 const _ = require('lodash');
 const Query = require('./Query');
 const match_phrase = require('../lib/leaf/match_phrase');
+const turf = require('@turf/turf');
+
+
+function createParentIdShould(layer, ids) {
+  // create an object initialize with terms.'parent.locality_id' (or whatever)
+  // must use array syntax for 2nd parameter as _.set interprets '.' as new object
+  return _.set({}, ['terms', `parent.${layer}_id`], ids);
+}
+
+function getLayersIdMap(vs) {
+  console.log('layers: ids', vs.var('input:layers:ids'));
+  console.log('layers', vs.var('input:layers'));
```

Comment:
Are these `console.log()` statements left in intentionally?

We have a `pelias/logger` package which we prefer to use since it supports log levels and makes logging more consistent, it also does a better job of writing Arrays/Objects compared to `console.log` which often truncates large nested structures.

Reply:
no, sorry, removed